### PR TITLE
qtxdg-mat: Rework exit codes

### PR DESCRIPTION
**Pull request by luis-pereira**
An asked --help must return EXIT_SUCCESS.
Any option passed to an non existent module must return EXIT_FAILURE.

### CODE REVIEW

File: src/tools/mat/qtxdg-mat.cpp [hunk 1]
```diff
@@ -95,9 +95,27 @@ int main(int argc, char *argv[])
     // Find out the positional arguments.
     parser.parse(QCoreApplication::arguments());
     const QStringList args = parser.positionalArguments();
-    const QString command = args.isEmpty() ? QString() : args.first();
-    bool cmdFound = false;
+    parser.parse(QCoreApplication::arguments());
```
Comment (tsujan) [inline src/tools/mat/qtxdg-mat.cpp:98]: Is there a reason to repeat it here? `parser.parse()` was called above, before a `const` method.

Comment (luis-pereira) [reply src/tools/mat/qtxdg-mat.cpp]: No, just a leftover typo. Fixed.
Thanks.

Comment (luis-pereira) [reply src/tools/mat/qtxdg-mat.cpp]: No, just a leftover typo. Fixed.
Thanks.

**Review by tsujan — APPROVED** [review]
GTM

**Pull request merged**